.babelrc

```diff
@@ -1,5 +1,12 @@
 {
-  "presets": ["env", "react"],
+  "presets": [
+    ["env", {
+      "targets": {
+        "chrome": 60
```

Comment:
do you think chrome is enough for this env? I imagine firefox supports the same stuff right?

Reply:
I'll add FF too :)